### ISSUE TITLE
Create selector functions for redux data

### ISSUE THREAD

**Issue by frozenhelium** [IFRCGo/go-frontend#862]
Instead of directly accessing the data from app state (similar to how we have action creators)
eg:
```
const mapStateToProps = (state, props) => ({
  projects: projectsSelector(state),
})

export default connect(mapStateToProps, mapDispatchToProps)(Component);
```
So that it's easier to refactor the variables used to represent project in the state

**Comment by batpad** [comment]
I agree.

cc @necoline 

@frozenhelium for this, I think it's fine to start new things following this pattern, and we'll ticket out doing this across the code-base for when we refactor.

**Comment by necoline** [comment]
I agree that this would be a good practice to incorporate moving forward.
